packages/api-clients/open-api/bffApi.yml

```diff
@@ -13535,6 +13535,8 @@ components:
           type: boolean
         activeDescriptor:
           $ref: "#/components/schemas/CompactDescriptor"
+        isDelegable:
+          type: boolean
```

Comment:
This is an Eservice in a list displaying compact info, I wouldn't return this field here unless explicitly requested. I would add it only to the calls where we also added `isSignalHubEnabled`, that don't return compact stuff. See: #1109 as a reference:
- both for BFF and for API GW (which seems to be missing here)
- both for GET and for Create calls (we must add it to the create seed)
- also for what concerns the eservice import / export -> it must be added there as well (same PR as reference)

The difference from that PR should be that here we also want to be able to edit it after the Eservice is published, meaning it should also be added to the seed of the corresponding call (discussion on Slack).

Could you then also update the Bruno calls and add some screens of local e2e tests?